### ISSUE TITLE
Sample should play before waveform is rendered

### ISSUE THREAD

**Issue by ViktorNova** [rhetr/surplus#5]
I have been finding Surplus extremely useful!

Waveform display had never worked for me until the latest round of updates, but Surplus seems to wait for the waveform to be drawn before playing the sample which, as you put it, slows everything down, and makes the process of going through banks of hundreds of samples to find the perfect snare a much more tedious chore.

It would be cool if the sample played first, and  the waveform drawing took place afterward, and would only complete if the user doesn't go to a new sample. This way it wouldn't slow down sample browsing

**Comment by rhetr** [comment]
good point, i'm still in the process of learning how to draw waveforms more efficiently...in the meantime you can edit the "Show Waveform" property in the config file to 0 (it should be at $HOME/.config/surplus/config)

**Comment by rhetr** [comment]
this is sort of fixed in a8869130f20fbb2fae92f1c3b58ebac3e732957a, I'm loading the waves on a separate thread now so it doesn't interfere too much with gui interaction. 

the main problem right now is that if you you hold down the up/down key and go through like over a hundred samples surplus will crash. I'm still figuring out a fix for this..
